### PR TITLE
Add support for generating PowerShell cmdlets using pre-built SDK assembly and specification.

### PR DESCRIPTION
**Pull request by bmanikm**
- Added support for generating PowerShell cmdlets using pre-built SDK assembly and specification.
- Please refer `New-PSSwaggerModule.md` file for details on new parameters being added.

PS: This PR doesn't include the support for downloading the SDK assembly and it's dependencies from NuGet.org.

### CODE REVIEW

File: docs/commands/New-PSSwaggerModule.md [hunk 1]
```diff
@@ -11,15 +11,29 @@ PowerShell command to generate the PowerShell commands for a given RESTful Web S
...
+### SpecificationPath (Default)
 ```
 New-PSSwaggerModule -SpecificationPath <String> -Path <String> -Name <String> [-Version <Version>]
  [-DefaultCommandPrefix <String>] [-Header <String[]>] [-UseAzureCsharpGenerator] [-NoAssembly]
  [-PowerShellCorePath <String>] [-IncludeCoreFxAssembly] [-InstallToolsForAllUsers] [-TestBuild]
  [-SymbolPath <String>] [-ConfirmBootstrap]
 ```
 
-### SwaggerURI
+### SdkAssemblyWithSpecificationPath
```
Comment (brywang-msft) [inline docs/commands/New-PSSwaggerModule.md]: >SdkAssemblyWithSpecificationPath [](start = 4, length = 32)

What's the plan to get Models namespace?

File: Tests/PSSwaggerScenario.Tests.ps1 [hunk 2]
```diff
@@ -846,3 +846,218 @@ Describe "Header scenario tests" -Tag @('Header','ScenarioTest')  {
         }
     }
 }
+
+Describe "Pre-compiled SDK Assmebly scenario tests" -Tag @('SDKAssembly','ScenarioTest')  {    
+    # As of today 9/7/2017, PSSwagger is not supported on PowerShell Core.
+    if((Get-Variable -Name PSEdition -ErrorAction Ignore) -and ('Core' -eq $PSEdition)) {
+        return
+    }
```
Comment (brywang-msft) [inline Tests/PSSwaggerScenario.Tests.ps1]: Ideally: generalize the helper function that takes care of core vs. full. No reason why these tests shouldn't run for core as well.

File: docs/commands/New-PSSwaggerModule.md (hunk 1 above)
Comment (bmanikm) [reply docs/commands/New-PSSwaggerModule.md]: ee36770

File: Tests/PSSwaggerScenario.Tests.ps1 (hunk 2 above)
Comment (bmanikm) [reply Tests/PSSwaggerScenario.Tests.ps1]: ee36770

**Pull request merged**